packages/react-components/src/atoms/Dropdown.js

```diff
@@ -72,30 +83,38 @@ const Dropdown = (props) => {
   return (
     <div className={user_type === "employee" ? "employee-select-wrap" : "select-wrap"} style={{ ...props.style }}>
       {/* <div className={userType === "employee" ? "select-wrap-emp" : "select-wrap"} style={{ ...props.style }}> */}
-      <div className={dropdownStatus ? "select-active" : "select"}>
-        <TextField
-          setFilter={setFilter}
-          forceSet={forceSet}
-          selectedVal={
-            selectedOption
-              ? props.t
-                ? props.t(props.optionKey ? selectedOption[props.optionKey] : selectedOption)
-                : props.optionKey
-                ? selectedOption[props.optionKey]
-                : selectedOption
-              : null
-          }
-          filterVal={filterVal}
-          // onClick={dropdownOn}
-          dropdownDisplay={dropdownOn}
-          freeze={props.freeze ? true : false}
-        />
-        <ArrowDown onClick={dropdownSwitch} />
-      </div>
+      {hasCustomSelector && (
+        <div className={`margin-right-30 cp margin-top-6 ${props.showArrow ? "flex-right" : ""}`} onClick={dropdownSwitch}>
```

Comment:
can we pragmatically render different classes rather than this?

Reply:
sure will update